demo/plugins.txt

```diff
@@ -0,0 +1,37 @@
+ace-editor:1.0.1
+authentication-tokens:1.2
+branch-api:1.3
+cloudbees-folder:5.2.2
+credentials:1.25
+docker-commons:1.3.1
+docker-workflow:1.3
+durable-task:1.7
+git-client:1.19.5
+git-server:1.6
+git:2.4.2
+github:1.14.0
+github-api:1.72
+github-branch-source:1.3
+handlebars:1.1
+javadoc:1.3
+jquery-detached:1.2
+junit:1.9
+momentjs:1.1
```

Comment:
:ant: This demo is actually using `matrix-auth`, so you should explicitly declare it as 1.3.2, rather than using the bundled version which is incompatible with the specified version of `cloudbees-folder`.